src/main/kotlin/eu/europa/ec/eudi/openid4vci/AuthorizeIssuance.kt

```diff
@@ -26,11 +26,14 @@ import java.time.Instant
  * for preparing the authorization request
  * @param state the state which was sent with the
  * authorization request
+ * @param identifiersSentAsAuthDetails The list of the offer's [CredentialConfigurationIdentifier]s that are or will be communicated to
+ * authorization server as detailed authorizations, part of a Rich Authorization Request.
  */
 data class AuthorizationRequestPrepared(
     val authorizationCodeURL: HttpsUrl,
     val pkceVerifier: PKCEVerifier,
     val state: String,
+    val identifiersSentAsAuthDetails: List<CredentialConfigurationIdentifier> = emptyList(),
```

Comment:
Let's avoid default value of empty list.

In any case this is populated by the lib.

Reply:
Done